src/components/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactChild, ReactFragment, ReactPortal } from "react";
+import getGreetingTime from "utils/getGreetingTime";
+
+function Footer(props: {
+  t: (
+    arg0: string,
+    arg1: { date: Date; context: string },
+  ) => boolean | ReactChild | ReactFragment | ReactPortal | null | undefined;
+}) {
+  return <div>{props.t("footer.date", { date: new Date(), context: getGreetingTime() })}</div>;
+}
+
+export default Footer;
```

Comment:
to nie jest komponent klasowy

Reply:
Przepraszam zmęczenie, funkcyjny xD

Reply:
nie ma sensu przekazywać w propsach tego t, normalnie użyj useTranslation()

Reply:
poprawione :)